### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by andrewsutjahjo**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+with open(REPO_ROOT / "README.md", encoding="utf-8") as f:
+    README = f.read()
+
+REQUIREMENTS = ["pandas"]
+
+setup(
+    name="phoenix-scrape",
+    version=versioneer.get_version(),
+    description="Lightweight framework for structured and repeatable model validation",
```
Comment (DBCerigo) [inline setup.py]: Should change

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    long_description=README,
+    long_description_content_type="text/markdown",
+    author="datavaluepeople",
+    author_email="opensource@datavaluepeople.com",
+    url="https://github.com/datavaluepeople/phoenix-scrape",
+    license="GNU",
```
Comment (DBCerigo) [inline setup.py]: Ultranit:  `GNU General Public License v3 (GPLv3)`

File: .github/workflows/make.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, '3.10']
```
Comment (DBCerigo) [inline .github/workflows/make.yml]: Nit: why the `'`s for 3.10?

File: .github/workflows/make.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        make install
+    - name: Run make
+      run: |
+        make
+    - name: Upload coverage to Codecov
+      if: ${{ matrix.python-version == 3.9 }}
```
Comment (DBCerigo) [inline .github/workflows/make.yml]: Ultranit: Do codecov on not a middle version, I would suggest the lowest version.

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```
Comment (DBCerigo) [inline .github/workflows/release.yml]: Shall we use 3.10?

**Review by DBCerigo — COMMENTED** [review]
Some minor tings

File: .github/workflows/make.yml (hunk 3 above)
Comment (andrewsutjahjo) [reply .github/workflows/make.yml]: if you don't do the `'` it parses it as a float and tries to install python `3.1`

File: .github/workflows/release.yml (hunk 5 above)
Comment (andrewsutjahjo) [reply .github/workflows/release.yml]: Not too sure, I've always done latest stable release -  1 minor version, but we could bump it to 10

File: .github/workflows/make.yml (hunk 3 above)
Comment (DBCerigo) [reply .github/workflows/make.yml]: derp

File: .github/workflows/release.yml (hunk 5 above)
Comment (DBCerigo) [reply .github/workflows/release.yml]: On reflection, I think it's all g and doesn't actually matter for `release.yml` which we use as long as that version is also unit tested ay

File: setup.py (hunk 1 above)
Comment (andrewsutjahjo) [reply setup.py]: 06ee0417530d2abfc656440500817a33ff3e29a9

Comment (andrewsutjahjo) [reply setup.py]: 9d12ebc8d5db99ac6c59dc8d631f3c9c7196ba61

File: setup.py (hunk 2 above)
Comment (andrewsutjahjo) [reply setup.py]: 9d12ebc8d5db99ac6c59dc8d631f3c9c7196ba61

File: .github/workflows/make.yml (hunk 4 above)
Comment (andrewsutjahjo) [reply .github/workflows/make.yml]: 9d12ebc8d5db99ac6c59dc8d631f3c9c7196ba61

File: .github/workflows/make.yml (hunk 3 above)
Comment (andrewsutjahjo) [reply .github/workflows/make.yml]: [301bdfe](https://github.com/datavaluepeople/phoenix-scrape/pull/2/commits/301bdfea6991c6f14907586b7d7202aae945d65b)

File: .github/workflows/release.yml (hunk 5 above)
Comment (andrewsutjahjo) [reply .github/workflows/release.yml]: [b7e2df3](https://github.com/datavaluepeople/phoenix-scrape/pull/2/commits/b7e2df3e71c728f8dd9c28fbe779bdaffed11a06)

**Review by DBCerigo — APPROVED** [review]

**Pull request merged**